README.md

```diff
@@ -33,47 +35,49 @@ EOF
 
 Note: change web-proxy.domain.net to the name of your proxy machine and adapt as well the port used.
 
-`#` **`mkdir -p /etc/docker`**
+`#` **`sudo mkdir -p /etc/docker`**
 
-`#` **`echo '{"registry-mirrors": ["http://myregistry:5000"], "insecure-registries": ["http://myregistry:5000", "http://myregistry:5001"], "dns": ["8.8.8.8", "4.4.4.4"]}' > /etc/docker/daemon.json`**
+`#` **`sudo echo '{"registry-mirrors": ["http://myregistry:5000"], "insecure-registries": ["http://myregistry:5000", "http://myregistry:5001"], "dns": ["8.8.8.8", "4.4.4.4"]}' > /etc/docker/daemon.json`**
 
```

Comment:
This doesn't work with sudo (Redirections ). So the initial comment which was saying that you need to trun that as root (meaning sudo su -) and the # prompt are the reason why putting sudo everytwhere in front is not the solution IMHO

Reply:
Thanks @bcornec I have refined further. Please review again.